### ISSUE TITLE
Status changes to "Not added" after a project is committed

### ISSUE THREAD

**Issue by BCerki** [bcgov/cas-cif#1646]
Describe the Bug:

Probability (how likely the bug is to happen, scored from 1-5):
(For example, probability of 5 is something like "it happens to all users every time they log in."
Probability of 1 "only happens to certain users when a really specific and unlikely path is followed.")

Effect (how bad the bug is when it does happen, scored from 1-5):
(For example, effect of 5 is "the entire app crashes and makes it unusable for all users" or "the bug causes the wrong data to be saved, with critical information (e.g. payment) being affected."
Effect of 1 is "It makes some styling look a little bit weird.")

Steps to reproduce the behaviour:

Screenshots:
https://www.loom.com/share/83c823e99a974b868b85d312042ea19e
Additional information:

- Browser: [e.g. chrome, safari]
- Version: [e.g. 22]
- Additional context

**Comment by suhafa** [comment]
@nanyangpro could you confirm how amendment type should appear in this scenario?

**Comment by pbastia** [comment]
I think it should be a list of checkboxes, just like in the creation process. Do we want to follow the same pattern with an "Update" button @nanyangpro ? I'd be in favour of just applying it on click, since it's a super low-consequence one, and has zero side effects

**Comment by BCerki** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Climate-Action-Secretariat-60ca4121764d710011481ca2/issues/bcgov/cas-cif/1646?planning-poker) with Zenhub @dleard @gurjmatharu @JoshLarouche @mikevespi @sam-warren @Sepehr-Sobhani

**Comment by nanyangpro** [comment]
Yes, @pbastia. It's a list of checkboxes. [Applying it on click](https://www.figma.com/proto/PzOY8RyQnVXnjZJdSv2AqQ/CIF---Internal-User-Portal?page-id=2289%3A2663&node-id=7283-1906&viewport=-470%2C1174%2C0.2&scaling=scale-down&starting-point-node-id=7283%3A1906&show-proto-sidebar=1) is the current design. :)
